src/ProjectTemplates/test/EmptyWebTemplateTest.cs

```diff
@@ -42,11 +42,15 @@ public async Task EmptyWebTemplateAsync()
 
             using (var aspNetProcess = Project.StartBuiltProjectAsync())
             {
+                Assert.False(aspNetProcess.Process.HasExited, ErrorMessages.GetErrorMessage("Run built project", Project, aspNetProcess.Process);
```

Comment:
Considering all the places we're copying this let's just put it inside AssertOk.

Reply:
Nope, the asserts are better of at the test method level so that:
1. You see a very specific error message about what test failed and why.
2. You get the exact line of code where the test blew up instead of having to dig through the call stack.

DRY is good for product code, not for tests.

Reply:
Hitting these endpoints is unlikely to have anything to do with why the process has stopped, and if that's what we're testing for oughtn't these checks run after we actually do something?

Reply:
This is checking that the process didn't exit early. Like when you do dotnet run and there is a framework mismatch or similar.